url_encoder/plugin.spec.yaml

```
@@ -0,0 +1,70 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_encoder
+title: URL Encoder
+description: Encode and decode URLs
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+cloud_ready: true
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_utils
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- url
+- encoder
+- decoder
+- utilities
+hub_tags:
+  use_cases: [data_utility]
+  keywords: [url, encoder, decoder, utilities]
+  features: []
+actions:
+  encode:
+    title: Encoder
```

Would recommend to make the action's intention a little more clear

```suggestion
    title: Encode URL
```